src/JustSaying/Messaging/Channels/SubscriptionGroups/SubscriptionConfigBuilder.cs

```diff
@@ -69,9 +93,9 @@ public SubscriptionConfigBuilder WithDefaultPrefetch(int prefetch)
 
         /// <summary>
         /// Specifies the default maximum number of messages that may be processed at once by a <see cref="ISubscriptionGroup"/>.
-        /// Defaults to Environment.ProcessorCount * 4
+        /// Defaults to Environment.ProcessorCount * 4.
```

Comment:
I still think it's better to explain this in words, rather than have the raw code copied verbatim into the comments.
If everyone else prefers this, then it should use a `<see>` tag to reference the property or be in a code tag.